conference/src/main/java/com/kyriba/conference/management/api/dto/TopicDto.java

```diff
@@ -23,4 +24,11 @@
   @ApiModelProperty(value = "Author")
   @NotBlank
   private String author;
+
+
+  public TopicDto(Topic topic)
```

Comment:
Probably this DTO should be in domain model and creation logic moved to the entity

Reply:
probably yes, but I'll left it as is for a while

Reply:
This is up to you and your team. ;)